elasticapm/instrumentation/packages/dbapi2.py

```diff
@@ -123,6 +123,8 @@ def extract_signature(sql):
     :param sql: the SQL statement
     :return: a string representing the signature
     """
+    if isinstance(sql, bytes):
+        sql = sql.decode()
```

Comment:
It might make sense to use `elasticapm.utils.encoding.force_text` here, as it will handle encoding errors somewhat gracefully